integration/test_containers.py

```diff
@@ -20,11 +20,11 @@ class TestContainers(IntegrationTestCase):
     def test_get(self):
         """A container is fetched by name."""
         name = self.create_container()
-        self.addCleanup(self.delete_container, name)
 
         container = self.client.containers.get(name)
 
         self.assertEqual(name, container.name)
+        self.addCleanup(self.delete_container, name)
```

Comment:
Why did you move this? If the test fails, the cleanup task won't be added and thus won't be cleaned up.